Source/Renderer/AutomaticUniforms.js

```diff
@@ -1588,6 +1588,20 @@ define([
             getValue : function(uniformState) {
                 return uniformState.shadowMap.lightPositionEC;
             }
+        }),
+
+        /**
+         * An automatic GLSL uniform representing the size of a shadow map texel in the [0,1] range.
+         *
+         * @alias czm_shadowMapTexelSize
+         * @glslUniform
+         */
+        czm_shadowMapTexelSize : new AutomaticUniform({
```

Comment:
"Size" is confusing.  Maybe call this "step size" or "delta"  For a blast from the past, see https://github.com/AnalyticalGraphicsInc/cesium/compare/depthPostFX